1.10.11/alpine3.10/Dockerfile

```diff
@@ -0,0 +1,123 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.10
+LABEL maintainer="Astronomer <humans@astronomer.io>"
+
+ARG ORG="astronomer"
+ARG VERSION="1.10.11-1.*"
+ARG SUBMODULES="all, statsd, elasticsearch"
+ARG AIRFLOW_MODULE="astronomer_certified[${SUBMODULES}]==$VERSION"
+ARG REPO_BRANCH=master
+
+LABEL io.astronomer.docker=true
+LABEL io.astronomer.docker.distro="alpine"
+LABEL io.astronomer.docker.module="airflow"
+LABEL io.astronomer.docker.component="airflow"
+LABEL io.astronomer.docker.airflow.version="1.10.11"
+LABEL io.astronomer.docker.ac.version="${VERSION}"
+
+ENV AIRFLOW_HOME="/usr/local/airflow"
+ENV PYTHONPATH=${PYTHONPATH:+${PYTHONPATH}:}${AIRFLOW_HOME}
+
+ARG ASTRONOMER_USER="astro"
+ARG ASTRONOMER_GROUP="astro"
+ENV ASTRONOMER_USER=${ASTRONOMER_USER}
+ENV ASTRONOMER_GROUP=${ASTRONOMER_GROUP}
+
+RUN addgroup -S ${ASTRONOMER_GROUP} \
+	&& adduser -S -G ${ASTRONOMER_GROUP} ${ASTRONOMER_USER}
+
+COPY include/humans@astronomer.io.rsa.pub /etc/apk/keys
+
+# Make pip look at our pip repo too, and force it to install these specific
+# versions when ever it installs a module.
+COPY include/pip.conf /etc/pip.conf
+COPY include/pip-constraints.txt /usr/local/share/astronomer-pip-constraints.txt
+
+# Install packages
+RUN echo https://github.com/astronomer/ap-airflow/raw/${REPO_BRANCH}/alpine-packages/3.10/repo >> /etc/apk/repositories \
+  && apk update \
+	&& apk add --no-cache --virtual .build-deps \
+		build-base \
+		cyrus-sasl-dev \
+		freetds-dev \
+		freetype-dev \
+		krb5-dev \
+		libffi-dev \
+		libxml2-dev \
+		libxslt-dev \
+		linux-headers \
+		python3-dev \
+		musl-dev \
+		postgresql-dev \
+		tzdata \
+	&& apk add --no-cache \
+		bash \
+		ca-certificates \
+		cyrus-sasl \
+		cython3 \
+		freetds \
+		krb5-libs \
+		py3-bcrypt \
+		py3-cassandra-driver'>='3 \
+		py3-cryptography \
+		py3-fastavro \
+		py3-grpcio \
+		py3-mysqlclient \
+		py3-numpy-dev \
+		py3-numpy \
+		py3-pandas \
+		py3-pip \
+		py3-psycopg2 \
+		py3-pycryptodome \
+		py3-pynacl \
+		py3-typed-ast \
+		python3 \
+		tini \
+	&& apk add --upgrade \
+		openssl \
+	&& update-ca-certificates \
+	&& cp /usr/share/zoneinfo/UTC /etc/localtime \
+	&& pip3 install --no-cache-dir --upgrade snowflake-connector-python==1.9.1 \
+	&& pip3 install --no-cache-dir "${AIRFLOW_MODULE}" \
+	&& pip3 install --no-cache-dir "https://github.com/astronomer/astronomer-airflow-scripts/releases/download/v0.0.4/astronomer_airflow_scripts-0.0.4-py3-none-any.whl" \
+	&& pip3 install --no-cache-dir "astronomer-fab-security-manager~=1.2, >=1.2.2" \
+	&& apk del .build-deps py3-numpy-dev \
+	&& ln -sf /usr/bin/python3 /usr/bin/python \
+	&& ln -sf /usr/bin/pip3 /usr/bin/pip \
+	&& sed -i -e '\#https://github.com/astronomer/#d' /etc/apk/repositories \
+	&& sed -i -e 's/^run_as_user = .*/run_as_user = 100/g' /usr/lib/python3.7/site-packages/airflow/config_templates/default_airflow.cfg
```

Comment:
@schnie We change the default UID for alpine image here to `100` 

The tests are passing :)

Reply:
I think I'd do this in a separate layer.

Also, we should probably set rbac=True default at the same time (Or have we done that in our wheels already?)

Reply:
`rbac=True` is done in our Helm chart: https://github.com/astronomer/airflow-chart/blob/master/templates/configmap.yaml#L34

Reply:
and moved the `sed` to separate layer